src/tools/pkcs11-tool.c

```diff
@@ -5363,6 +5455,18 @@ static int test_decrypt(CK_SESSION_HANDLE sess)
 		printf("No OpenSSL support, unable to validate decryption\n");
 #else
 		for (n = 0; n < num_mechs; n++) {
+			switch (mechs[n]) {
+			case CKM_RSA_PKCS:
+			case CKM_RSA_PKCS_OAEP:
+			case CKM_RSA_X_509:
+			//case CKM_RSA_PKCS_TPM_1_1:
+			//case CKM_RSA_PKCS_OAEP_TPM_1_1:
```

Comment:
Do we need these comments here? Do you have a pkcs11 that presents them? Does it work?

Reply:
These two mechanisms are defined in the "official" header files that I am using and described in the 2.40 specifications.
They don't exist in the header files of your project, this is why I have commented them.
My pkcs11 won't support them, so I won't be able to test.
I can remove these two lines.

Reply:
2 lines removed @https://github.com/OpenSC/OpenSC/pull/1600/commits/d25fbe3cecc55857541ad773899235e227409c47